assets.json

```diff
@@ -12189,61 +12189,59 @@
+"textures/gTexturePortraitLuigi.rgba16.ci8.png": {"meta":{"dims":[32,32], "pal":"0x1A6D8"}, "offsets": {"us":["0x132B50", "0x1BAD8"]}},
+"textures/gTexturePortraitPeach.rgba16.ci8.png": {"meta":{"dims":[32,32], "pal":"0x1A8D8"}, "offsets": {"us":["0x132B50", "0x1BED8"]}},
+"textures/gTexturePortraitToad.rgba16.ci8.png": {"meta":{"dims":[32,32], "pal":"0x1AAD8"}, "offsets": {"us":["0x132B50", "0x1C2D8"]}},
+"textures/gTexturePortraitYoshi.rgba16.ci8.png": {"meta":{"dims":[32,32], "pal":"0x1ACD8"}, "offsets": {"us":["0x132B50", "0x1C6D8"]}},
+"textures/gTexturePortraitDonkeyKong.rgba16.ci8.png": {"meta":{"dims":[32,32], "pal":"0x1AED8"}, "offsets": {"us":["0x132B50", "0x1CAD8"]}},
+"textures/gTexturePortraitWario.rgba16.ci8.png": {"meta":{"dims":[32,32], "pal":"0x1B0D8"}, "offsets": {"us":["0x132B50", "0x1CED8"]}},
+"textures/gTexturePortraitBowser.rgba16.ci8.png": {"meta":{"dims":[32,32], "pal":"0x1B2D8"}, "offsets": {"us":["0x132B50", "0x1D2D8"]}},
+"textures/132B50_1D6D8.rgba16.ci8.png": {"meta":{"dims":[64,32], "pal":"0x1B4D8"}, "offsets": {"us":["0x132B50", "0x1D6D8"]}},
 
-"textures/132B50_1DED8_tlut.rgba16.png": {"meta":{"dims":[16,16]}, "offsets": {"us":["0x132B50", "0x1DED8"]}},
-"textures/132B50_1E0D8_tlut.rgba16.png": {"meta":{"dims":[16,16]}, "offsets": {"us":["0x132B50", "0x1E0D8"]}},
-"textures/132B50_1E2D8_tlut.rgba16.png": {"meta":{"dims":[16,16]}, "offsets": {"us":["0x132B50", "0x1E2D8"]}},
-"textures/132B50_1E4D8_tlut.rgba16.png": {"meta":{"dims":[16,16]}, "offsets": {"us":["0x132B50", "0x1E4D8"]}},
-"textures/132B50_1E6D8_tlut.rgba16.png": {"meta":{"dims":[16,16]}, "offsets": {"us":["0x132B50", "0x1E6D8"]}},
-"textures/132B50_1E8D8_tlut.rgba16.png": {"meta":{"dims":[16,16]}, "offsets": {"us":["0x132B50", "0x1E8D8"]}},
-"textures/132B50_1EAD8_tlut.rgba16.png": {"meta":{"dims":[16,16]}, "offsets": {"us":["0x132B50", "0x1EAD8"]}},
-"textures/132B50_1ECD8_tlut.rgba16.png": {"meta":{"dims":[16,16]}, "offsets": {"us":["0x132B50", "0x1ECD8"]}},
-"textures/132B50_1EED8_tlut.rgba16.png": {"meta":{"dims":[16,16]}, "offsets": {"us":["0x132B50", "0x1EED8"]}},
-"textures/132B50_1F0D8_tlut.rgba16.png": {"meta":{"dims":[16,16]}, "offsets": {"us":["0x132B50", "0x1F0D8"]}},
-"textures/132B50_1F2D8_tlut.rgba16.png": {"meta":{"dims":[16,16]}, "offsets": {"us":["0x132B50", "0x1F2D8"]}},
-"textures/132B50_1F4D8_tlut.rgba16.png": {"meta":{"dims":[16,16]}, "offsets": {"us":["0x132B50", "0x1F4D8"]}},
-"textures/132B50_1F6D8_tlut.rgba16.png": {"meta":{"dims":[16,16]}, "offsets": {"us":["0x132B50", "0x1F6D8"]}},
-"textures/132B50_1F8D8_tlut.rgba16.png": {"meta":{"dims":[16,16]}, "offsets": {"us":["0x132B50", "0x1F8D8"]}},
-"textures/132B50_1FAD8_tlut.rgba16.png": {"meta":{"dims":[16,16]}, "offsets": {"us":["0x132B50", "0x1FAD8"]}},
-"textures/132B50_1FCD8_tlut.rgba16.png": {"meta":{"dims":[16,16]}, "offsets": {"us":["0x132B50", "0x1FCD8"]}},
+"textures/gTLUTItemWindowNone.rgba16.png": {"meta":{"dims":[16,16]}, "offsets": {"us":["0x132B50", "0x1DED8"]}},
```

Comment:
I'm wondering if gItemFrame would be a more accurate name than window.

Reply:
Maybe? I don't know what to call it really and decided to just copy what the user manual says.